src/poojap/Shop.java

```diff
@@ -0,0 +1,121 @@
+package poojap;
+
+public class Shop {
+	
+	int qMaggie;
+	int qDosa ;
+	int qPouches; 
+	int qPanipuri;  
+	int qMasala;
+
+	void setInitial( int orderMaggy, int orderDosa, int orderPouches, int orderPanipuri, int orderMasala) {
+	
+		qMaggie = orderMaggy;
+		qDosa = orderDosa;
+		qPouches = orderPouches;
+		qPanipuri = orderPanipuri;
+		qMasala = orderMasala;
+		
+	 System.out.println("Customer purchased :" +qMaggie+ " Maggi packets " +qDosa+" Dosa Packets " +qPouches+" oil Packets " +qPanipuri+ " Panipuri packets " +qMasala+ " Masala packets");
+
+	}
+		void customerPurchase(int orderMaggy, int orderDosa, int orderPouches, int orderPanipuri, int orderMasala) {
+			
+	       System.out.println("Customer purchased :" +orderMaggy+ " Maggi packets " +orderDosa+" Dosa Packets " +orderPouches+" oil Packets " +orderPanipuri+ " Panipuri packets " +orderMasala+ " Masala packets");
+
+
+		if(qMaggie >=orderMaggy) 
+			qMaggie = qMaggie - orderMaggy;
+		    System.out.println("Maggie:Running Out of stock");
+		
+		if(qDosa >=orderDosa)
+		   qDosa = qDosa - orderDosa;
+	   	   System.out.println("Dosa:Running Out of stock");
+    
+
+		if(qPouches >= orderPouches) 
+		   qPouches = qPouches - orderPouches;
+	   	   System.out.println("oil:Running Out of stock");
+
+	   
+		if(qPanipuri>= orderPanipuri) 
+		   qPanipuri = qPanipuri - orderPanipuri;
+	   	   System.out.println("Panipuri:Running Out of stock");
+
+
+		if(qMasala >=orderMasala)
+		  qMasala = qMasala - orderMasala;
+	  	  System.out.println("Masala:Running Out of stock");
+
+	}
+	void outOfStock() {
+
+        boolean flag = false;
```

Comment:
you dont need to use boolean value & you write multiple if condition and for all those if conditions you have one common boolean falg which is wrong logically.

Refer below code: https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/blob/main/src/kajol/ShopStockDetails.java